client/src/components/MainPage/QuestionsPage/QuestionCard/index.jsx

```diff
@@ -7,39 +7,35 @@ const Card = (props) => {
   const { values, section } = props;
   return (
     <ul className="box">
-      {values && values.map((element) => {
-        const {
-          question, id, username, translations, date,
-        } = element;
-        return (
-          <li key={id} className="box__card">
-            <Link to={`/main/${section}/questions/${id}`}>
-              <div>{question}</div>
+      {values
+        && values.map((element) => {
+          const {
+            question, id, username, translations, date,
+          } = element;
```

Comment:
@HemaSAli You can destructure right in the function parenthesis, and save a line or two of code 😀
`&& values.map(({ question, id, username, translations, date }) => {`

Reply:
thanks Tom , I edited it